src/pages/MonthlyGood/MonthlyGoodSetupPageControl.vue

```diff
@@ -385,28 +384,8 @@ export default {
 	inject: ['apollo'],
 	apollo: {
 		query: pageQuery,
-		preFetch(config, client, { route }) {
-			return client.query({ query: userInfoQuery })
-				.then(({ data }) => {
-					const userId = _get(data, 'my.userAccount.id');
-					if (!userId) {
-						throw new Error('activeLoginRequired');
-					}
-				})
-				.then(() => {
-					return client.query({ query: pageQuery });
-				})
-				.catch(e => {
-					if (e.message.indexOf('activeLoginRequired') > -1) {
-						// Force a login when active login is required
-						return Promise.reject({
-							path: '/ui-login',
-							query: { doneUrl: route.fullPath }
-						});
-					}
-					// Log other errors
-					console.error(e);
-				});
+		preFetch(config, client) {
```

Comment:
since this is just calling pageQuery, this doesn't need to be a function and can just be `preFetch: true`